### PR TITLE
DT-1135: rename DLUHC -> MHCLG

### PR DESCRIPTION
**Pull request by hugh-emerson**
**Release** Terraform + app. No downtime

### CODE REVIEW

File: terraform/production/main.tf [hunk 1]
```diff
@@ -77,7 +77,7 @@ module "auth_service" {
     from_name        = "Delta System"
     from_address     = "delta@datacollection.levellingup.gov.uk"
     reply_to_name    = "DLUHC Digital Services"
```
Comment (keithkennedy) [inline terraform/production/main.tf:79]: These should also be updated.

File: terraform/production/main.tf [hunk 2]
```diff
@@ -33,7 +33,7 @@ locals {
   cloudwatch_log_expiration_days = 731
   dclg_access_group_notification_settings = {
     enabled                     = true
-    additional_recipient_emails = ["Group-DLUHCDeltaNotifications@softwire.com", "delta-notifications@levellingup.gov.uk", "DELTAadmin@levellingup.gov.uk", "DELTAStatSupport@levellingup.gov.uk"]
+    additional_recipient_emails = ["Group-DLUHCDeltaNotifications@softwire.com", "delta-notifications@communities.gov.uk", "DELTAadmin@levellingup.gov.uk", "DELTAStatSupport@levellingup.gov.uk"]
```
Comment (keithkennedy) [inline terraform/production/main.tf:36]:
```suggestion
    additional_recipient_emails = ["Group-DLUHCDeltaNotifications@softwire.com", "delta-notifications@communities.gov.uk", "DELTAadmin@communities.gov.uk", "DELTAStatSupport@communities.gov.uk"]
```

**Review by keithkennedy — APPROVED** [review]
Some additional changes suggested, otherwise looks good.

File: terraform/production/main.tf (hunk 1 above)
Comment (hugh-emerson) [reply terraform/production/main.tf:79]: From address can't be updated yet, but yeah may as well update the name.

**Pull request merged**